console/src/main/webapp/WEB-INF/i18n/application.properties

```diff
@@ -24,6 +24,9 @@ editUserDetailsForm.fieldset.userDetails=User details
 editUserDetailsForm.fieldset.credentials=Credentials
 editUserDetailsForm.changePassword.link=Change Password
 editUserDetailsForm.success=User details updated successfully
+editUserDetailsForm.organisation=Organisation
```

Comment:
Organization

Reply:
Both are corrects. We use an «s» in the console.